TestAssets/TestPackages/dotnet-dependency-context-test/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-preview1-*",
+  "version": "1.0.0-preview2-*",
```

Comment:
It's probably because of this rule:

https://github.com/NuGet/NuGet.Client/blob/d316bf56e4f55e94d166b31a0189464015b8a844/src/NuGet.Core/NuGet.Versioning/SemanticVersionFactory.cs#L136-L160

`            // 0 is fine, but 00 is not. `